test_scripts/Defects/8_2/3882/3882_9_two_apps_different_hmi_types_in_ptu_default_section.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3882
+---------------------------------------------------------------------------------------------------
+-- Description: SDL sends `UI.ChangeRegistration` with updated HMI type after PTU with replaced HMI type
+--  in default section for several registered apps
+--
+-- Steps:
+-- 1. HMI and SDL are started
+-- 2. App1 is registered with <HMI type 1>
+-- 3. App2 is registered with <HMI type 2>
+-- 4. App1 is activated
+-- 5. PTU is performed with <HMI type 3> for default permissions
+-- SDL does:
+-- - send two UI.ChangeRegistration("appHMIType" = { <HMI type 3> }) to HMI for each app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Defects/8_2/3882/common')
+
+--[[ Local Functions ]]
+local function updFuncWraper(pHMItype)
+  local function updFunc(pTbl)
+    pTbl.policy_table.app_policies.default.AppHMIType = pHMItype
+    if pTbl.policy_table.app_policies[common.getPolicyAppId(1)] then
+      pTbl.policy_table.app_policies[common.getPolicyAppId(1)] = "default"
+    elseif pTbl.policy_table.app_policies[common.getPolicyAppId(2)] then
+      pTbl.policy_table.app_policies[common.getPolicyAppId(2)] = "default"
+    end
```

Comment:
```suggestion
      pTbl.policy_table.app_policies[common.getPolicyAppId(1)] = "default"
      pTbl.policy_table.app_policies[common.getPolicyAppId(2)] = "default"
```
Seems we don't update the policy for App2 since the first condition is true